elytron/pom.xml

```diff
@@ -320,6 +320,7 @@
                             <excludes>
                                 <exclude>*-1.*.xml</exclude>
                                 <exclude>*-4.*.xml</exclude>
+                                <exclude>elytron-subsystem-*.xml</exclude>
```

Comment:
Once this exclusion is added what does the XML validation test?  In the past this has been very useful to detect discrepancies in the schema.

Reply:
@SoniaZaldana I think this question is still outstanding?

Reply:
@darranl My apologies, I updated the test and forgot to ping you. In order to avoid bypassing the validation test for the overall subsystem schema, I created a separate test class just for resolving expressions with a separate XML file. Then, only this new XML file would be excluded from the schema validation.